.github/workflows/publish-immutable-actions.yml

```
@@ -17,4 +17,4 @@ jobs:
         uses: actions/checkout@v4
       - name: Publish
         id: publish
-        uses: actions/publish-immutable-action@0.0.3
+        uses: actions/publish-immutable-action@0.0.4
```

There is a typo in the current string that Dependabot fixed in its PR
```suggestion
        uses: actions/publish-immutable-action@v0.0.4
```

---

Interesting, just noticed because VSCode also complains about the missing `v`, since it tries to resolve a tag that doesn't exist (without the `v`)

---

Hi